packages/cfd/src/Containers/mt5-mobile-redirect-option.tsx

```diff
@@ -19,7 +19,7 @@ const MT5MobileRedirectOption = ({ mt5_trade_account }: TMT5MobileRedirectOption
             mobile_url = window.location.replace(getMobileAppInstallerURL({ mt5_trade_account }) as string);
         }, 1500);
 
-        if (!isSafariBrowser()) {
+        if (!isSafariBrowser() || (isSafariBrowser() && /iPhone OS 17/.test(navigator.userAgent))) {
```

Comment:
Is this required in this PR. Also since this is a change in current cfd, do we need QA to test this change?

Reply:
not required in this PR, thanks for catching

reverting this file change